docs/api.yml

```diff
@@ -0,0 +1,226 @@
+openapi: "3.0.0"
+info:
+  title: RT Dashboard API
+  version: 1.0.0
+
+paths:
+  /history.json:
+    get:
+      summary: List history entries
+      responses:
+        200:
+          description: |
+            Return object containting list of rows and groups of history entries
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  rows:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        group:
+                          type: string
+                        subgroup:
+                          type: number
+                        start:
+                          type: number
+                        end:
```

Comment:
`end` might not be present https://github.com/djangsters/rt-dashboard/blob/7008ffdb55db848f5882957de16725ba5748c279/rt_dashboard/history.py#L100

Reply:
`end` is not a required field